charts/radar-output/templates/configmap.yaml

```diff
@@ -43,6 +43,13 @@ data:
         {{- if .azure.readTimeout }}
         readTimeout: {{ .azure.readTimeout | int }}
         {{- end }}
+      index:
+        {{- if .index.fullSyncInterval }}
```

Comment:
Why the `if` statement if we already have a default value for this?
If both `if`s are off, then the chart will be invalid since there will be and empty `index:` key.

Reply:
I am just doing the same as for all other values?

Reply:
Ok, I removed the if statements